apps/src/p5lab/AnimationPicker/AnimationPickerListItem.jsx

```diff
@@ -38,55 +38,34 @@ class AnimationPickerListItem extends React.Component {
       selected
     } = this.props;
     const {loaded, hover} = this.state;
-    const rootStyle = [styles.root, !label && styles.noLabel];
-
-    const thumbnailStyle = [
-      styles.thumbnail,
-      icon && styles.thumbnailIcon,
-      animationProps && {
-        display: loaded ? 'block' : 'none'
-      }
-    ];
 
-    const labelStyle = [
-      styles.label,
-      icon && styles.labelIcon,
-      animationProps && {
-        display: loaded ? 'block' : 'none'
-      }
-    ];
     const iconImageSrc = category
       ? `/blockly/media/p5lab/animation-previews/category_${category}.png`
       : '';
 
-    const thumbnailStyleWithHover = [
-      thumbnailStyle,
-      hover && styles.multiSelectBorder,
-      hover && styles.hoverBorder,
-      selected && styles.selectBorder
-    ];
-
     const multiSelectIconClassName = `fa ${
       selected ? 'fa-check' : 'fa-plus'
     } fa-2x`;
-    const multiSelectIconStyle = [
-      styles.multiSelectIcon,
-      hover && styles.hoverIcon,
-      selected && styles.selectIcon
-    ];
 
     return (
       <div
-        style={rootStyle}
+        className={classNames(style.root, !label && style.noLabel)}
         onFocus={() => this.setState({hover: true})}
         onBlur={() => this.setState({hover: false})}
         onMouseEnter={() => this.setState({hover: true})}
         onMouseLeave={() => this.setState({hover: false})}
       >
         <button
-          style={thumbnailStyleWithHover}
           onClick={onClick}
-          className={category}
+          className={classNames(
+            style.thumbnail,
+            icon && style.thumbnailIcon,
+            hover && style.multiSelectBorder,
+            hover && style.hoverBorder,
+            selected && style.selectBorder,
```

Comment:
now that we're using CSS, i wonder if this state is necessary anymore or if we can replace it with `:hover` and `:active` styles

Reply:
I actually reverted this today because it seemed to not be working as expected. Happy to take another look.

Reply:
eh, it's also an improvement we can make later. if it isn't easily fixable, i'd say it's outside the scope of this refactor since we can get rid of radium without this additional change